src/main/java/com/tournament/cup/play/TeamsListFour.java

```diff
@@ -0,0 +1,46 @@
+package com.tournament.cup.play;
+
+import com.tournament.cup.details.Squad;
+import lombok.AccessLevel;
+import lombok.experimental.FieldDefaults;
+import org.springframework.stereotype.Repository;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Repository
+@FieldDefaults(level = AccessLevel.PRIVATE)
+public class TeamsListFour {
```

Comment:
This class looks very similat to TeamListEith. The difference is the content of the teams. If you want to have predefined set of teams, please use the same class parameterized with different arguments. Otherwise it leads to duplication